Shared/Localizations/ru.lproj/Localizable.strings

```diff
@@ -8,21 +8,21 @@
 
 // MARK: - Generic
 // Unknown, default string for applications
-"UNKNOWN" = "Неизвестный";
+"UNKNOWN" = "Неизвестно";
 // Default string
 "DEFAULT" = "По умолчанию";
 // Copy provided string in a menu
 "COPY" = "Копировать";
 // Delete item
 "DELETE" = "Удалить";
 // Cancel alert action
-"CANCEL" = "Отменить";
+"CANCEL" = "Отмена";
 // Done action
 "DONE" = "Готово";
 // Dismiss action
-"DISMISS" = "Отклонить";
+"DISMISS" = "Закрыть";
 // Dismiss 2 action
-"LAME" = "ОК";
+"LAME" = "Понятно";
```

Comment:
"Понятно" is more informal and conversational, making it inappropriate for most interfaces. it might be used in casual contexts, such as games or applications with a specific speech style, but for standard user interfaces and in iOS, "OK" should not be translated and remains the best choice for clarity and familiarity